starter-code/src/components/ProductRow.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import data from '../data.json';
+
+
+class ProductRow extends Component {
+  render() {
+    return (
+      <tr>
+        <td style={{color:this.props.stocked ? "" : "red"}}>{this.props.name}</td>
```

Comment:
L'expression ternaire peut être remplacée par `color:this.props.stocked || "red"`